### PR TITLE
Update lumi.json

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: domains/lumi.json [hunk 1]
```diff
@@ -6,5 +6,6 @@
   },
   "record": {
     "A": ["76.76.21.21"]
+    "TXT" : ["dh=e00668e963ac416881a25e0bb5f87429a5eb0792"]
```
Comment (CuteDog5695) [inline domains/lumi.json:9]: Wrong way to use discord TXT verification, please go here to see how to do it: https://is-a.dev/docs/discord-verification/

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/lumi.json (hunk 1 above)
Comment (CuteDog5695) [reply domains/lumi.json:9]: Btw the actual problem was because you didn't add a comma at the end of the A line

Comment (itsFatlum) [reply domains/lumi.json:9]: I actually completely over read that I need to create a new file. Sorry!

File: domains/_discord.lumi.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "itsFatlum",
+        "email": ""
```
Comment (CuteDog5695) [inline domains/_discord.lumi.json:4]: I recommend adding your discord ID in the "discord" string, this should be how you should do it here:
```json
        "username": "itsFatlum",
        "email": "",
        "discord": "enterdiscordid"
```

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by itsFatlum** [comment]
You should add the Discord part on the docs website since it's missing.

**Review by iostpa — APPROVED** [review]

**Pull request merged**